app/src/main/java/uw/hcrlab/kubi/MainActivity.java

```diff
@@ -46,100 +47,88 @@ protected void onCreate(Bundle savedInstanceState) {
         //Notice: this is how each activity will get the robot and connect it to the robot face
         getWindow().addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
         setContentView(R.layout.activity_main);
-        robot = Robot.getInstance((RobotFace)findViewById(R.id.face), this);
-    }
 
-    /*
-    Called after your activity has been stopped, prior to it being started again.
-    Always followed by onStart()
-     */
-    @Override
-    protected void onRestart() {
-        Log.i(TAG, "Restarting Main Activity ...");
-        super.onRestart();
-    }
+        eyesView = (SurfaceView) findViewById(R.id.eyesView);
+
+        Uri video = Uri.parse("android.resource://" + getPackageName() + "/" + R.raw.look_left_then_right);
+        mp = new MediaPlayer();
+        mp.setOnPreparedListener(new MediaPlayer.OnPreparedListener() {
+            @Override
+            public void onPrepared(MediaPlayer mediaPlayer) {
+                mediaPlayer.setOnSeekCompleteListener(new MediaPlayer.OnSeekCompleteListener() {
+                    @Override
+                    public void onSeekComplete(MediaPlayer mediaPlayer) {
+                        mediaPlayer.setOnVideoSizeChangedListener(new MediaPlayer.OnVideoSizeChangedListener() {
+                            @Override
+                            public void onVideoSizeChanged(MediaPlayer mediaPlayer, int width, int height) {
+                                Log.d(TAG, "Video size: " + width + ", " + height);
+                            }
+                        });
+
+                        mediaPlayer.setDisplay(eyesView.getHolder());
+                    }
+                });
+
+                mediaPlayer.seekTo(0);
+            }
+        });
+
+        try {
+            mp.setDataSource(this, video);
+            mp.prepareAsync();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```

Comment:
This was just some test code to see what was possible. Last night Sahil sent me a sent of mp4's of eye animations, so now I can work on making the whole eye system work.